packages/amplication-code-gen-types/src/plugin-events.ts

```diff
@@ -61,6 +62,7 @@ export type Events = {
   [EventNames.CreateServerPackageJson]?: PluginEventType<CreateServerPackageJsonParams>;
   [EventNames.CreateAdminUIPackageJson]?: PluginEventType<CreateAdminUIPackageJsonParams>;
   [EventNames.CreateServerAppModule]?: PluginEventType<CreateServerAppModuleParams>;
+  [EventNames.CreateConnectMicroservices]?: PluginEventType<CreateConnectMicroservicesParams>;
```

Comment:
Let's make an update to the docs with an example of how to add a microservice. :)

Reply:
https://github.com/amplication/amplication/issues/6704